### PR TITLE
docs: fixed issue #2325 and added additional step  for Gatsby and Next.js storefronts

### PR DESCRIPTION
**Pull request by ayushthe1**
* What - Docs : added additional step for changing the directory to `my-medusa-storefront` for Gatsby and Next.js storefronts #2325 

* Why - In step 2 of `Installation` for [Gatsby](https://docs.medusajs.com/starters/gatsby-medusa-starter/#installation) and [Next.js](https://docs.medusajs.com/starters/nextjs-medusa-starter/#installation) Storefront ,it is mentioned to "change to the directory" but without that being portrayed in the command.
* How - Following the guideline in the issue made the changes to `medusa/docs/content/tutorial/0-set-up-your-development-environment.mdx`
* Testing - it's a markup file, so should not require anything other than manual testing.

### CODE REVIEW

File: docs/content/starters/gatsby-medusa-starter.md [hunk 1]
```diff
@@ -30,6 +30,10 @@ gatsby new my-medusa-storefront https://github.com/medusajs/gatsby-starter-medus
 
 2\. Change to the newly created directory `my-medusa-storefront` and rename the template environment variable file to use environment variables in development:
 
+```bash
+cd my-medusa-storefront
+```
+
 ```bash
 mv .env.template .env.development
 ```
```
Comment (shahednasser) [inline docs/content/starters/gatsby-medusa-starter.md:39]:
~~~suggestion
```bash
cd my-medusa-storefront
mv .env.template .env.development
```
~~~

File: docs/content/starters/nextjs-medusa-starter.md [hunk 2]
```diff
@@ -22,13 +22,17 @@ This document assumes you already have a Medusa server installed. If you don’t
...
-1. Change to the newly created directory `my-medusa-storefront` and rename the template environment variable file to use environment variables in development:
+2. Change to the newly created directory `my-medusa-storefront` and rename the template environment variable file to use environment variables in development:
+
+```bash
+cd my-medusa-storefront
+```
 
 ```bash
 mv .env.template .env.local
 ```
```
Comment (shahednasser) [inline docs/content/starters/nextjs-medusa-starter.md:33]:
~~~suggestion
```bash
cd my-medusa-storefront
mv .env.template .env.local
```
~~~

File: docs/content/starters/nextjs-medusa-starter.md [hunk 3]
```diff
@@ -22,13 +22,17 @@ This document assumes you already have a Medusa server installed. If you don’t
...
 
-1. Make sure the Medusa server is running, then run the local Next.js server:
+3. Make sure the Medusa server is running, then run the local Next.js server:
```
Comment (shahednasser) [inline docs/content/starters/nextjs-medusa-starter.md:35]: Remove this change as it is being covered by another PR

File: docs/content/starters/nextjs-medusa-starter.md [hunk 4]
```diff
@@ -22,13 +22,17 @@ This document assumes you already have a Medusa server installed. If you don’t
 npx create-next-app -e https://github.com/medusajs/nextjs-starter-medusa my-medusa-storefront
 ```
 
-1. Change to the newly created directory `my-medusa-storefront` and rename the template environment variable file to use environment variables in development:
+2. Change to the newly created directory `my-medusa-storefront` and rename the template environment variable file to use environment variables in development:
```
Comment (shahednasser) [inline docs/content/starters/nextjs-medusa-starter.md:25]: Remove the number change as it is being covered by another PR

**Review by shahednasser — CHANGES_REQUESTED** [review]
Thank you @ayushthe1 for your work, I've added some comments for fixes.

**Review by ayushthe1 — COMMENTED** [review]
Fixed the issues as commented.

**Comment by ayushthe1** [comment]
@shahednasser i fixed the issues as commented by you. Can you please review it.

**Review by shahednasser — APPROVED** [review]

**Comment by shahednasser** [comment]
Thanks! @ayushthe1 🚀

**Comment by ayushthe1** [comment]
Thanks @shahednasser  for your support .Please give me more issues to contribute. I would love to work more on this project.

**Pull request merged**